R/zchunk_L221.en_supply.R

```diff
@@ -92,295 +105,661 @@ module_energy_L221.en_supply_DISABLED <- function(command, ...) {
     L132.ag_an_For_Prices <- get_data(all_data, "L132.ag_an_For_Prices")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+
+    # Supplysector information for upstream energy handling sectors
+    A21.sector %>%
+      write_to_all_regions(c(LEVEL2_DATA_NAMES[["Supplysector"]]), has_traded = TRUE,
+                           GCAM_region_names = GCAM_region_names) -> L221.Supplysector_en
+
+    # Supplysector table that indicates to the model to create solved markets for them
+    # The traded markets tend to be a good candidate to solve explicitly since they tie together
+    # many solved markets
+    A21.sector %>%
+      write_to_all_regions(c("region", "supplysector", "traded"), has_traded=TRUE,
```

Comment:
I unreservedly and completely apologize for being so picky, but: ` = ` (spaces around the equals)